integration-enabler-java/build.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-    compile(project(':common-service-core'))
+    compile(project(':apiml-common'))
```

Comment:
the apiml-common module has classes used by Spring. Integration-enabler-java should be in pure Java without depending on Spring. No?